server_api.py

```diff
@@ -9,15 +9,160 @@
 import bcrypt
 import json
 import tornado
+import time
 
 from oauth2client import client
 from oauth2client import tools
 from oauth2client.file import Storage
 import httplib2
 
 from apiclient import discovery
+#from arrange_schedule import find_acticity
+#from arrange_schedule import edit_schedule
 import datetime
 #
+def check_bluetooth_mode_available():
+    file_dir = "static/user_data"
+    file_name = "setting.txt"
```

Comment:
I think that setting.txt should be put under a dir "bluetooth" rather than put in "static/user_data", please also create an directory add ".keep" file for committing.